.buildkite/pipelines/pull_request/scout_tests.yml

```diff
@@ -8,8 +8,5 @@ steps:
       SCOUT_CONFIGS_SCRIPT: '.buildkite/scripts/steps/test/scout_configs.sh'
     retry:
       automatic:
-        # No retry when Scout configs fail
-        - exit_status: 10
-          limit: 0
```

Comment:
It has no affect, status code 10 is not returned in this step, but in the one that runs tests